projects/js-packages/webpack-config/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.2.1 - 2021-11-19
```

Comment:
We should have something in the section here. Did changelogger somehow leave this empty, or did you manually delete the "Updated package dependencies" entry?

Reply:
The changelogger left it empty; the only changelog I manually edited was Boost's. 

I've inserted a changelog entry there manually for updating project dependencies in 3e4b92a1b. Thanks for picking up on this.